static/app/components/core/slider/index.chonk.tsx

```diff
@@ -33,9 +34,9 @@ export function Slider({formatLabel = passthrough, ref, ...props}: SliderProps)
           props.onChange?.(newValue, e);
         }}
       />
-      {props.disabled ? null : (
+      {props.disabled || !formattedLabel ? null : (
```

Comment:
@isabellaenriquez the falsy check here wont display the label if the value is 0, we should limit it to only null and undefined :)

Reply:
good point!